### PR TITLE
Add Qmailtoaster, IMP, Music Browser, Darwin Streaming Server, Kimai

### PR DESCRIPTION
**Pull request by quinncomendant**
I added Qmailtoaster, IMP, Music Browser, Darwin Streaming Server, Kimai. I also added the APSLv2 license, which is used by the Darwin Streaming Server.

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [✔️] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language` ``

- [✔️] Your additions are ordered alphabetically.
- [✔️] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [✔️] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [✔️] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [✔️] You have searched the repository for any relevant issues or PRs.
- [✔️] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 entries with syntax error(s).</td>
    </tr>
  </tbody>
</table>

#### Syntax Issues

 Name | Entry
----|----------------------
Music Browser |   * [Music Browser](https://github.com/henrik242/musicbrowser) - Browser and streamer for your music collection. It is runs on most operating systems, and is light enough to run flawlessly on NAS devices.  `GNU GPL v3` `PHP` 
 

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -9,5 +9,5 @@ Please join us on [chat](https://chat.awesome-selfhosted.com) to clarify any que
  * Non-[Free](https://en.wikipedia.org/wiki/Free_software) software must be marked `⊘ Proprietary` next to the title, and added to [non-free.md](non-free.md).
  * The `⚠` character must be inserted for software that relies on a third party service the user does not control, e.g ``[Name](http://homepage/) `⚠` - Short description...``.
  * Check current [issues](https://github.com/Kickball/awesome-selfhosted/issues) and [Pull Requests](https://github.com/Kickball/awesome-selfhosted/pulls), as well as [closed issues/PRs](https://github.com/Kickball/awesome-selfhosted/pulls?utf8=%E2%9C%93&q=is%3Aclosed+).
- * The list of contributors can be updated with `make contrib`. Several checks are available: `make noexternallink; make missinglicenselanguage; make nofullstop` 
+ * The list of contributors can be updated with `make contrib`. Several checks are available: `make noexternallink; make nolicenselanguage; make nofullstop; make longdescriptions; make listlicenses; make syntaxerrors` or run all with `make checks`.
```
Comment (nodiscc) [inline .github/CONTRIBUTING.md]: Indeed, thanks

File: README.md [hunk 2]
```diff
@@ -274,6 +274,7 @@ _simple deployment of a mail server, e.g. for inexperienced or impatient admins.
   * [Mail-in-a-Box](https://mailinabox.email/) - Turns any Ubuntu server into a fully functional mail server with one command. ([Source Code](https://github.com/mail-in-a-box/mailinabox)) `CC0` `Shell`
   * [Modoboa](http://modoboa.org/en/) - Modoboa is a mail hosting and management platform including a modern and simplified Web User Interface. ([Source Code](https://github.com/tonioo/modoboa)) `MIT` `Python`
   * [Postal](https://github.com/atech/postal) - Postal is a complete and full featured mail server for use by websites and web servers. `MIT` `Ruby`
+  * [Qmailtoaster](http://www.qmailtoaster.com/) - Stable, full-featured, easy-to-install mail server based on qmail. ([Source Code](https://github.com/QMailToaster/)) `multiple` `Linux`
```
Comment (nodiscc) [inline README.md]: Capital M to `Multiple`

File: README.md [hunk 3]
```diff
@@ -321,6 +322,7 @@ _Mailing lists servers and mass mailing software - one message to many recipient
   * [Roundcube](https://roundcube.net) - Browser-based IMAP client with an application-like user interface. ([Source Code](https://github.com/roundcube/roundcubemail/)) `GPLv3` `PHP`
   * [SquirrelMail](http://squirrelmail.org) - Another browser-based IMAP client. ([Source Code](https://sourceforge.net/p/squirrelmail/code/HEAD/tree/)) `GPLv2` `PHP`
   * [WebMail Lite](http://www.afterlogic.org/webmail-lite) - Web-based IMAP Mail client. ([Source Code](https://github.com/afterlogic/webmail-lite)) `GPLv3` `PHP`
+  * [IMP](https://www.horde.org/apps/imp/) - HORDE application that provides webmail access to IMAP and POP3 accounts. ([Demo](http://demo.horde.org/), [Source Code](https://www.horde.org/download/imp)) `GNU GPL v2` `PHP`
```
Comment (nodiscc) [inline README.md]: Please order the section alphabetically

Comment (nodiscc) [reply README.md]: License should be `GPLv2`

File: README.md [hunk 4]
```diff
@@ -775,11 +777,13 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [mpd](http://www.musicpd.org/) - Daemon to remotely play music, stream music, handle and organize playlists. Many clients available. `GPLv2` `C++`
   * [mStream](http://mstream.io/) - Music streaming server with GUI management tools. Runs on Mac, Windows, and Linux. ([Demo](https://darncoyotes.mstream.io/), [Source Code](https://github.com/IrosTheBeggar/mStream)) `GPLv2` `Nodejs`
    * [ympd](http://www.ympd.org/) - Standalone MPD Web GUI written in C, utilizing Websockets and Bootstrap/JS. ([Source Code](https://github.com/notandy/ympd)) `GPLv2` `C`
+  * [Music Browser](https://github.com/henrik242/musicbrowser) - Browser and streamer for your music collection. It is runs on most operating systems, and is light enough to run flawlessly on NAS devices.  `GNU GPL v3` `PHP`
```
Comment (nodiscc) [inline README.md]: Please order the section alphabetically

Comment (nodiscc) [reply README.md]: License should be `GPLv3`

File: README.md [hunk 5]
```diff
@@ -809,6 +813,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [GeneWeb](https://geneweb.tuxfamily.org/wiki/GeneWeb) - GeneWeb is an open source genealogy software written in OCaml. It comes with a Web interface and can be used off-line or as a Web service. ([Demo](https://demo.geneweb.tuxfamily.org/gw7/), [Source Code](https://github.com/geneweb/geneweb)) `GPLv2` `OCAML`
   * [How Secure Is My Password](https://github.com/howsecureismypassword/hsimp) - Rather than just saying a password is "weak" or "strong", How Secure is My Password? lets your users know how long it would take someone to crack their password. ([Demo](https://howsecureismypassword.net/)) `MIT` `Javascript`
   * [google-webfonts-helper](https://github.com/majodev/google-webfonts-helper) `⚠` - A Hassle-Free Way to Self-Host Google Fonts. Get eot, ttf, svg, woff and woff2 files + CSS snippets. ([Demo](https://google-webfonts-helper.herokuapp.com/)) `MIT` `Nodejs`
+  * [Kimai](http://www.kimai.org/) - Simple time and project tracking. ([Demo](http://www.kimai.org/demo/), [Source Code](https://github.com/kimai/kimai)) `GNU GPL v3` `PHP`
```
Comment (nodiscc) [inline README.md]: License should be `GPLv3`

**Comment by quinncomendant** [comment]
I've applied the changes you suggested with 6112793. I also fixed some inconsistent indentation in the lists (all lists are now indented by two spaces).

**Comment by nodiscc** [comment]
@quinncomendant Thanks, but please send the indentation changes as a separate Pull Request

**Comment by quinncomendant** [comment]
Pushed this again without the indentation fixes.

**Comment by quinncomendant** [comment]
I created [issue 1201](https://github.com/Kickball/awesome-selfhosted/issues/1201) to discuss the list indentation.

**Comment by quinncomendant** [comment]
Removed a double-space from the Music Browser line that was causing tests to fail.

File: README.md (hunk 5 above)
Comment (nodiscc) [reply README.md]: I think you missed this ^
Thanks for the fixes and for raising #1201

**Comment by quinncomendant** [comment]
Horde (not to be confused with Horde Groupware) is a web app dev framework. Alone, it doesn't offer any end-user functionality, and shouldn't be on the selfhosted list at all.

IMP is the webmail client. One of its dependencies is the Horde framework.

They also make [many other web apps](https://www.horde.org/apps/), including:

- Kronolith (calendar)
- Turba (address book)
- Nag (todo list)
- Mnemo (notes)
- Whups (ticketing system)
- Ansel (photo mgmt)
- Wicked (wiki)
- …and umpteen others.

All of them require the Horde framework. 

They offer a package called *Horde Groupware* which includes several of their apps together in one package (Kronolith, Turba, Nag and Mnemo).

Another package called *Horde Groupware Webmail Edition* includes everything in the above plus IMP.

To be fair, all their apps should be listed in selfhosted in their proper categories. I included only IMP because it is their flagship, and is the only one I've use (it's complex, but good).

**Pull request merged**